content/getting-started/registering.md

```diff
@@ -4,8 +4,71 @@ prev: "/getting-started/"
+You have proved yourself worthy. You have proved yourself committed and willing to learn and improve your skills. You are allowed to enter CodeWars dojo. To get your account created, fill in a couple of details, or use your GitHub account to authenticate on CodeWars:
+
+![enlist](./img/registering_05_enlist.png)
+
+Great, your account is created!
+
+**Important note:** CodeWars will send you an email to confirm your email account. Go to your inbox and click on confirmation link in the message titled `"Confirm your account... Dojo Access Link"`, received from CodeWars. Until you do this, you will be allowed to train and solve tasks, but some actions will be restricted or unavaliable for you. For example, you won't be able to create discourse posts. If you cannot find the email, check `Spam` folder of your inbox! If you do not confirm your account right now, you can always ask CodeWars to resend you the confirmation email by clicking dedicated button in your account panel (we will get to your account panel soon).
+
+## Your training routine
+
+There's only one last step you need to do before you start solving tasks: you need to set up your training routine, so CodeWars would know what to suggest you and how to drive you through your learning process. Don't worry, you just need to answer a few simple questions:
+
+![initial training setup](./img/registering_06_initial_training_setup.png)
+
+ - **What is your programming experience?** - select any option which you feel describes you the best. If you are making your first steps in the world of programming, just pick "Learning to Program". If you are a seasoned programmer who wants to train their coding-fu, improve their skills, and probably learn somethng new, you can choose "Senior Developer".
+ - **Would you like to receive our newsletter with weekly challenges?** - choose "Yes" to receive a weekly email with some suggested tasks to solve. Email contains a choice of CodeWars problems of every difficulty, so you will be always able to find something what would suit you. Participating in Weekly Coding Challenge is a great way to keep yourself motivated! If you do not want to receive the email, just pick "No".
+ - **Choose the language you wish to train on** - you can select from a large list of languages available on CodeWars. You can select one, but if you want, you can select more. During your training, CodeWars suggests you tasks which are available in languages you choose here. If you are not sure if you want to select some language or not, don't worry! You can always switch to any language available for trained task. As I want to improve my functional programming skills, and as I heard, Haskell is a very nice FP language, I am going to select this one too.
+
+Remember that if you would like to change your answers later, you will be always able to do so in your [Training Setup panel](https://www.codewars.com/trainer/setup). If you answered all above questions, just click `SAVE` and you are done!
```

Comment:
Remember that if you would like to change your answers later, you will always be able to do so in your [Training Setup panel](https://www.codewars.com/trainer/setup). If you answered all above questions, just click `SAVE` and you are done!